pysoa/test/plugins/fixtures.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import (
+    absolute_import,
+    unicode_literals,
+)
+
+import importlib
+import os
+
+import pytest
+
+
+@pytest.fixture(scope='session')
```

Comment:
I had a thunk. Why session scope? This makes it difficult to override just one fixture on a per-module basis, because then you also have to override all the other fixtures that depend on it. I would argue `scope='module'` or even `scope='function'` is more appropriate.

Reply:
`scope` is kind of a misnomer, it just defines the lifetime of the fixture. `session` means that the fixture will be evaluated at most once per test run, but if the fixture itself lives in a test module, it will only be available to tests in that module, and will only override equivalently named fixtures in the scope of that module.